Sources/ProjectSpec/ProjectReference.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ProjectReference.swift
+//  ProjectSpec
+//
+//  Created by Yuta Saito on 2019/10/15.
+//
+
+import Foundation
+import JSONUtilities
+
+public struct ProjectReference {
+    public let name: String
```

Comment:
@giginet I actually disagree with this change. Mutability of structs can be locked off wherever that struct is referenced, it doesn't need to be immutable an instance variable level. Also mutating structs is not a potentally dangerous operation like it is with classes, as you're not mutating data out from under anyone. Because of this the rest of the project spec is all vars for easy editing.

Reply:
I understood your opinion. anything will be fine.

@kateinoigakukun  Could you revert these changes 🙏